sig-docs/localization-subproject/README.md

```diff
@@ -0,0 +1,105 @@
+# Localization Subproject
+
+The Kubernetes Localization Subproject is owned by [SIG Docs](https://github.com/kubernetes/community/tree/master/sig-docs) and run by the [Subproject leads](#leadership).
+
+The Kubernetes documentation has been localized into several different languages. Each localization is owned by a team of individuals that speaks the language.  A localization team is a self-sufficient community responsible for prioritizing work, translating content, and reviewing and merging PRs into the kubernetes/website repo for their localization. 
+
+The Localization Subproject's aim is to provide a place for localization teams to share ideas and standardize processes across all localizations. This page covers documentation, processes, and roles for the Kubernetes Localization teams and the Localization Subproject.
+
+## Meetings
+
+Regular Localization Meeting: The first Monday of the month at 15:00 UTC (biweekly). [Convert it to your timezone](http://www.thetimezoneconverter.com/?t=15:00&tz=UTC)
```

Comment:
The addition of biweekly here confuses me, since this is a monthly meeting – am I interpreting this incorrectly?

Reply:
updated!